canonical/ubuntu.com
# Update copy on /openstack

## Done

- Update copy on /openstack based on [copydoc](https://docs.google.com/document/d/1eqHK0hc9v7Mn-XV36QRyNkJenTXgBQR_-RULKm5rZ5o/edit)
- formatting (changed commented)

## QA

- Go to https://ubuntu-com-13817.demos.haus/openstack and check the changes match the [copydoc](https://docs.google.com/document/d/1eqHK0hc9v7Mn-XV36QRyNkJenTXgBQR_-RULKm5rZ5o/edit)

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-10586

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)
